lib/index.js

```diff
@@ -30,12 +30,8 @@ var defaults = {
       'ios > 6, android > 4.3, samsung > 3, chromeandroid > 50'
   },
   'postcss-easy-import': {
-    onImport: function(imported) {
-      // Update the watch task with the list of imported files
-      if (typeof global.watchCSS === 'function') {
-        global.watchCSS(imported);
-      }
-    }
+    transform: identity,
```

Comment:
I'd put faith in V8 taking care of optimisations around simple functions. Based on the various plugins I think it'd be a while before the bottle neck was the preprocessor.